emacs/radian.el

```diff
@@ -3257,6 +3257,22 @@ Return either a string or nil."
         (push (expand-file-name "bin" venv) exec-path))
       (apply func args))))
 
+(radian-use-package lsp-pyright
+  :after lsp
+  :hook ((python-mode . radian--lsp-pyright-discover-virtualenvs)
+         (python-mode . (lambda () (require 'lsp-pyright))))
+  :config
+  (defun radian--lsp-pyright-discover-virtualenvs ()
```

Comment:
Can we factor this out so that it only runs once when visiting a Python buffer, instead of twice (once for each of the two LSPs)?